scenarios/utils/job/job_utils.go

```diff
@@ -10,8 +10,38 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-// IsListedWithStatus Checks if job exists with status
-func IsListedWithStatus(cfg config.Config, appName, expectedStatus string, logger *log.Entry) (*models.JobSummary, error) {
+// GetLastPipelineJobWithStatus Checks if a last pipeline job exists with status
+func GetLastPipelineJobWithStatus(cfg config.Config, appName, expectedStatus string, logger *log.Entry) (*models.JobSummary, error) {
+	jobSummaries, err := getPipelineJobs(cfg, appName)
+	if err != nil {
+		return nil, err
+	}
+	lastJobSummary := jobSummaries[0]
+	if lastJobSummary.Status != expectedStatus {
+		return nil, fmt.Errorf("method GetLastPipelineJobWithStatus for application %s expected status '%s', but it received '%s'",
+			appName, expectedStatus, lastJobSummary.Status)
+	}
+	logger.Debugf("method GetLastPipelineJobWithStatus for application %s received expected status '%s'", appName, expectedStatus)
+	return lastJobSummary, nil
+}
+
+// GetAnyPipelineJobWithStatus Checks if any pipeline job exists with status
+func GetAnyPipelineJobWithStatus(cfg config.Config, appName, expectedStatus string, logger *log.Entry) (*models.JobSummary, error) {
+	jobSummaries, err := getPipelineJobs(cfg, appName)
+	if err != nil {
+		return nil, err
+	}
+	for _, jobSummary := range jobSummaries {
+		if jobSummary.Status == expectedStatus {
+			logger.Debugf("method GetAnyPipelineJobWithStatus for application %s received expected status '%s'", appName, expectedStatus)
```

Comment:
is it necessary to log to debug console?

Reply:
Good to have this option in prod